src/packages/elevator/elevator.taro.tsx

```
@@ -103,29 +98,35 @@
-        .selectAll(`.${classPrefix}-${uuid} .nut-elevator-item-${i}`)
-        .boundingClientRect()
-      // eslint-disable-next-line no-loop-func
-      query.exec((res: any) => {
-        if (res[0][0]) height += res[0][0].height
-        state.current.listHeight.push(height)
+    const query = createSelectorQuery()
+    query
+      .selectAll(`#${classPrefix}-${uuid} .${classPrefix}-list-item`)
+      .boundingClientRect()
+      .exec((rect = []) => {
+        if (rect[0] && rect[0].length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)